src/testdrive/src/action/postgres/connect.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use async_trait::async_trait;
+
+use crate::action::postgres::postgres_client;
+use crate::action::{Action, State};
+use crate::parser::BuiltinCommand;
+
+pub struct ConnectAction {
+    connection: String,
+    url: String,
+}
+
+pub fn build_connect(mut cmd: BuiltinCommand) -> Result<ConnectAction, String> {
+    let connection = cmd.args.string("connection")?;
```

Comment:
nit: here `connection` means "name" but in `postgres-execute` it means "name or URL". So perhaps a bit clearer to call it `name` here instead of `connection`? Then `name` means "name", `url` means "url", and `connection` means "name or url", which feels right to me.

Reply:
Thanks, agreed. Fixed.